api/lib/DBClient.js

```diff
@@ -4,21 +4,32 @@ const user = process.env.CLOUDANT_USER;
 const pw = process.env.CLOUDANT_PW;
 
 function DBClient() {
-    this._client = Cloudant({account:user, password:pw});
+    this._client = Cloudant({ account: user, password: pw });
 }
 
-DBClient.prototype.get = function(db, from, size) {
+DBClient.prototype.get = function (options) {
+    // The Cloudant api does the verification of the 'from' and 'size' already.
+    if (typeof options === "object") {
+        if (options.db) {
+            if (typeof options.db !== 'string') {
```

Comment:
You can simplify these 2 in a single if like this:
```
if (options.db && typeof options.db !== 'string')
```

You could even merge it with the if at line 12 as well, but, in my point of view, the separation from checking the type of options and checking db is good because we may want to validate other fields inside options in the future, so I think it's ok to maintain it.

Reply:
ok! thanks!